src/test/java/org/aksw/gerbil/annotator/decorator/ErrorCountingAnnotatorDecoratorTest.java

```diff
@@ -118,7 +118,7 @@ public static class SimpleTestDatasetConfig extends AbstractDatasetConfiguration
         private int size;
 
         public SimpleTestDatasetConfig(int size) {
-            super("test dataset", false, ExperimentType.ERec, null, null);
+            super("test dataset","Un Grouped", false, ExperimentType.ERec, null, null);
```

Comment:
Please use the static attribute that you defined further above.

Reply:
Done